content/project/board-election-process.adoc

```diff
@@ -15,188 +15,175 @@ More contributors might be added to the election committee if approved by the li
 The terms of office for these elected positions are:
 
-* Officer positions (1 year): December 03, 2020 to December 2, 2021
-* Governing board member (2 years): December 03, 2020 to December 2, 2022
+* Officer positions (one year): December 03, 2021 to December 2, 2022
+* Governing board member (two years): December 03, 2021 to December 2, 2023
 
 === References
 
-* link:/blog/2020/12/03/election-results[Election results]
-* link:/blog/2020/10/28/election-candidates[Election candidates]
-* link:/blog/2020/09/24/board-elections/[Election announcement]
+* link:/blog/2021/12/03/election-results[Election results]
+* link:/blog/2021/10/25/jenkins-elections/[Voter registration]
+* link:/blog/2021/09/20/election-period-opened/[Election announcement]
 
 === Key dates
 
-* **Sep 24** - Nominations open
-* **Sep 24** - Voting sign-up begins
-* **Oct 15** - Nominations deadline
-** After the deadline, the election committee will process nominations and notify the candidates
-* **Oct 26** or later - Election candidates are published, including personal statements
-* **Nov 10** - Voting begins
-** Email is distributed to all registered voters through link:https://civs.cs.cornell.edu/[Condorcet Internet Voting Service]
-* **Nov 08** - Voting sign-up is over
+* *October 20* - Nominations open
+* *October 20* - Voting registration begins
 ** During the registration period, the election committee will monitor the sign-ups and update the list of voters.
-* **Nov 27** - Voting ends, 11PM UTC
-* **Dec 03** - Results are announced and take effect.
+* *November 10* - Nominations deadline
+** After the deadline, the election committee will process nominations and notify the candidates.
+* *November 17* - Voting registration is closed
+* *November 17* - Election candidates and personal statements published
+* *November 17* - Voting begins
+** Email is distributed to all registered voters through link:https://civs.cs.cornell.edu/[Condorcet Internet Voting Service].
+* *December 02* - Voting ends, 11PM UTC
+* *December 07* - Results are announced
 
 === Elections Committee
 
-The 2020 elections are coordinated by the Jenkins Governance Board members who are not up for re-election this year:
-link:/blog/authors/slide_o_mix/[Alex Earl], link:/blog/authors/uhafner/[Ullrich Hafner], and link:/blog/authors/oleg_nenashev/[Oleg Nenashev].
+The elections are coordinated by the Jenkins Governance Board members who are not up for re-election this year.
 These contributors are responsible for managing the process, preparing the nominee list for elections, forming and verifying the voter list, and processing/announcing the election results.
 
-You can contact the election committee via mailto:jenkins-2020-elections@googlegroups.com[jenkins-2020-elections@googlegroups.com].
-Please use this email for any queries and feedback regarding the elections.
+You can contact the election committee via link:https://community.jenkins.io/g/election-committee[Jenkins Election Committee].
```

Comment:
we could enable email for that group if you wanted btw. Discourse has an email gateway.

Reply:
Thanks for the insight on that @halkeye, I was thinking it might be better to have one place to send the nominations so no "beams get crossed" so to speak. If the email has helped in years prior, I would be more than happy to update and implement the email.